nova/tests/unit/network/test_neutronv2.py

```diff
@@ -3439,9 +3423,6 @@ def _fake_list_ports(**search_opts):
                 list_ports_mock, list_networks_mock, show_quota_mock):
 
             self.api.validate_networks(self.context, requested_networks, 1)
-
-            self.assertEqual(len(list_port_values),
-                             len(list_ports_mock.call_args_list))
```

Comment:
I don't think that's the right fix. We should update the corresponding `list_port_values` instead of not even if checking it's called the first time. Maybe even compare the call args for that since call, i.e. make it a `assert_called_once_with()`